main.js

```diff
@@ -0,0 +1,276 @@
+let start = document.getElementById("start");
+let compStart = document.getElementById("compStart");
+let status = document.getElementById("status");
+let cellButton = Array.from(document.getElementsByClassName("cellButton"));
+let timer = document.getElementById("timer");
+let replay = document.getElementById("replay");
+
+//define variables
+let systemStatus = true; // whose turn it is
+let winStatus = false;
+let oStatus; // is player O a human or the computer
+let cell; // cell from within cellButton array
+let cellId; // actual div id from html
+let interval; // shows the timer count in seconds
+let playTimer; // the timer itself
+let userName; // name user chooses / if not defaults to X
+let XMoves = []; // creates array of moves by player X
+let OMoves = []; // creates array of moves by player O
+let possibleMoves = [
+  "cell-0",
+  "cell-1",
+  "cell-2",
+  "cell-3",
+  "cell-4",
+  "cell-5",
+  "cell-6",
+  "cell-7",
+  "cell-8",
+]; // list of all cells - updated throughout play so computer cannot guess duplicate play
```

Comment:
Great set up for all of these variables, everything is declared clearly and concisely at the top level. Great use of arrays to track board/cell/move changes. Super clear commenting of everything important.